markdown/dev/tutorials/pattern-design/testing-your-pattern/en.md

```diff
@@ -95,19 +97,20 @@ covered in this tutorial. It is left *as an exercise to the reader*.
 
 </Note>
 
-<Example 
-  sample 
-  part="bib" 
-  pattern="tutorial" 
-  settings={{ 
-    sample: { 
-      type: "option", 
-      option: "widthRatio" 
-    } 
-  }} 
+\<Example
```

Comment:
Linter removed indentation, which may or may not effect the rendering depending on which library you're using in the site